core/src/replay_stage.rs

```diff
@@ -393,8 +392,8 @@ impl ReplayTiming {
                     i64
                 ),
                 (
-                    "process_gossip_duplicate_confirmed_slots_elapsed",
-                    self.process_gossip_duplicate_confirmed_slots_elapsed as i64,
+                    "process_duplicate_confirmed_slots_elapsed",
+                    self.process_duplicate_confirmed_slots_elapsed as i64,
```

Comment:
Would the name change here break any dashboard etc?

Reply:
yeah dashboard would have to be updated, but that happens all the time. also this metric is pretty niche, would be surprised if anyone outside of us is actively tracking it.